### PR TITLE
HAAR-592: README updates

### PR DESCRIPTION
**Pull request by AlTurner-MOJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,10 +48,18 @@ The authorization code flow is also supported and can be tested via Postman usin
 ```bash
 client-id=test-client-id
 client-secret=test-secret
-access-token-url=localhost:8089/authorization-server/oauth2/token
-auth-url=localhost:8089/authorization-server/oauth2/authorize
+access-token-url=localhost:8089/oauth2/token
```
Comment (SimonMitchellMOJ) [inline README.md:51]: Does this need http:// at the beginning as well

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Review by karenmillermoj — COMMENTED** [review]
Not part of this update but I think if we're using the profile local-postgres it is running against a postgres db not HSQLDB ?
Also, rather than directing to the docker-compose-test file - would be good to just show the command itself.
If we can run in either H2 or Postgres, it would be nice to have 2 different sets of instructions

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**